crypto/ec/ecp_nistp521.c

```diff
@@ -1253,6 +1253,20 @@ static void point_add(felem x3, felem y3, felem z3,
     /* ftmp5[i] < 2^61 */
 
     if (x_equal && y_equal && !z1_is_zero && !z2_is_zero) {
+        /* This is obviously not constant-time but it will almost-never happen
```

Comment:
There's already a comment on top of the file about this, perhaps either adjust it to say "almost never" and point to this comment; or remove it in favour of this one?

Reply:
where's the other comment, in ```make_points_affine```?  I'll remove it or make that edit when I merge.

Reply:
Sorry, I meant "on top of the function". Above, on line 1159:

```
  * This function includes a branch for checking whether the two input points
  * are equal (while not equal to the point at infinity). This case never		  
  * happens during single point multiplication, so there is no timing leak for		  
  * ECDH or ECDSA signing. */
```

Reply:
I replaced this "This case never..." to end of comment with a "see comment below"